Failing allowed_kwargs test

I presume this is due to #331.

````
    def test_allowed_kwargs():
        """Test #332 where allowed_kwargs class attribute is unique for each subclass"""
        new_kwargs = ACATable.allowed_kwargs - ACACatalogTable.allowed_kwargs
        assert new_kwargs == {'call_args', 'version'}
    
        new_kwargs = FidTable.allowed_kwargs - ACACatalogTable.allowed_kwargs
>       assert new_kwargs == {'acqs'}
E       AssertionError: assert {'acqs', 'exc...'include_ids'} == {'acqs'}
E         Extra items in the left set:
E         'include_ids'
E         'exclude_ids'
E         Use -v to get the full diff
```

--- a/proseco/tests/test_catalog.py
+++ b/proseco/tests/test_catalog.py
@@ -29,7 +29,7 @@ def test_allowed_kwargs():
     assert new_kwargs == {'call_args', 'version'}
 
     new_kwargs = FidTable.allowed_kwargs - ACACatalogTable.allowed_kwargs
-    assert new_kwargs == {'acqs'}
+    assert new_kwargs == {'acqs', 'include_ids', 'exclude_ids'}
 
 
 @pytest.mark.skipif('not HAS_SC_ARCHIVE', reason='Test requires starcheck archive')

